Turkey/TestOutputFormat.cs

```diff
@@ -41,15 +46,26 @@ public async override Task AfterRunningTestAsync(string name, TestResult result,
                     switch (result)
                     {
                         case TestResult.Passed: resultOutput = "\u001b[32mPASS\u001b[0m"; break;
-                        case TestResult.Failed: resultOutput = "\u001b[31mFAIL\u001b[0m"; break;
+                        case TestResult.Failed: resultOutput = "\u001b[31mFAIL\u001b[0m"; failedTests.Add($"{string.Format("{0,-60}", name)}[{resultOutput}]\t({elapsedTime})"); break;
                         case TestResult.Skipped: resultOutput = "SKIP"; break;
                     }
                     Console.WriteLine($"[{resultOutput}]\t({elapsedTime})");
                 }
             }
 
+            public async override Task TestFailures()
```

Comment:
Any reason we don't use `AfterRunningAllTestsAsync` for this?

Reply:
Thanks for pointing this out.